### PR TITLE
Rewrite main page description

### PR DESCRIPTION
**Pull request by snappizz**
The overall idea here is to make the first impression more appealing to the computer music crowd. Also clearly delineate scsynth/sclang/scide right off the bat, because that distinction is fundamental to SC.

I don't have the exact numbers of UGens for core and sc3-plugins, please fix and replace the 666's. also the sclang description is unfinished.

if you have push access to this repo feel free to commit your changes directly to this branch.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -6,13 +6,19 @@ group: index
...
 <div class="jumbotron">
-    <p><strong>SuperCollider is an audio server, programming language, and IDE for sound synthesis and algorithmic composition.</strong></p>
 
-    <p>The language interpreter runs in a cross platform IDE (OS X/Linux/Windows) and communicates via Open Sound Control with one or more synthesis servers.
-    The SuperCollider synthesis server runs in a separate process or even on a separate machine so it is ideal for realtime networked music.</p>
+    <p>SuperCollider is a platform for audio synthesis and algorithmic composition, used by musicians, artists, and researchers working with sound. It is free and open source software available for Windows, Mac OS X, and Linux.</p>
+
+    <p>SuperCollider features three major components:
+      <ul>
+        <li>scsynth, a real-time audio server, forms the core of the platform. It features 666 unit generators ("UGens") for analysis, synthesis, and processing. Some of the audio techniques it supports include additive synthesis, subtractive, FM, granular, FFT, and physical modelling. You can write your own UGens in C++, and users have already contributed over 666 to the sc3-plugins repository.</li>
```
Comment (telephon) [inline index.md]: 666 is a joke that not everyone understands?

Comment (crucialfelix) [inline index.md]: Let's count them. I think it's good to present facts not vague statements.  We could add "and growing"  to keep it current sounding

Comment (telephon) [inline index.md]: yes, let's distribute the work: everyone counts a few and then we all get together and add up what we've got.

Comment (snappizz) [inline index.md]: (
    UGen.allSubclasses
    .asSet
    .remove(BEQSuite)
    .add(Mix)
    .add(PMOsc)
    .add(BLowPass4)
    .add(BHiPass4)
    .add(Silent)
    .collect(_.name)
    .collect(_.asString)
    .select(_.endsWith("UGen").not)
    .select(_.endsWith("Base").not)
    .select(_.beginsWith("Abstract").not)
    .size;
    )
    -> 406

Comment (crucialfelix) [inline index.md]: @snappizz wins

Comment (snappizz) [inline index.md]: I filtered out BinaryOpUGen and UnaryOpUGen, which are legitimate ugens. So the real count is 408, assuming I'm not missing any pseudo-ugens.

**Pull request merged**